Fenced_Frames_Ads_Reporting.md

```diff
@@ -196,6 +205,8 @@ function addBeaconData(element) {
 
 The beacon data will be in place by the time that the navigation starts. When the navigation commits, the automatic beacon will be sent out with event data set to "link1 was clicked.".
 
+`once` is optional, and, if empty, will default to false. If `once` is set to true, the automatic beacon will only be sent for the next `reserved.top_navigation` event. Beacons will not be sent for subsequent `reserved.top_navigation` events until `setReportEventDataForAutomaticBeacons` is invoked again. When used with a click handler, this can be used to send beacons only for specific top-level navigations, rather than for every top-level navigation.
```

Comment:
From reading this text I don't understand what use cases "once" satisfies.  I would have thought the very first navigation in a Fenced Frame would navigate the browser to a whole new top level page in which case the Fenced Frame (and any opportunity to report from there) is lost, so in essence all beacons added here are only sent once before top level navigation.  Can you add an example use case here to capture/explain when this feature is most useful?  The explanation of the use case could go immediately prior to the example I asked for in my first comment.

Reply:
Added a use case.